go/pkg/client/client.go

```diff
@@ -76,19 +85,31 @@ func NewClient(ctx context.Context, host string, port string, options ...ClientO
 
 	client.ticketFact = newTicketFactory()
 
-	client.sessionStub, err = newSessionStub(ctx, client)
+	client.flightStub, err = newFlightStub(client, host, port)
 	if err != nil {
 		client.Close()
 		return nil, err
 	}
 
-	client.consoleStub, err = newConsoleStub(ctx, client, opts.scriptLanguage)
+	cfgClient := configpb2.NewConfigServiceClient(grpcChannel)
+
+	if auth == "" {
+		auth = "Anonymous"
```

Comment:
Suggest moving magic string literals like this (and "http.session.durationMs") to some utility class so they can be defined in one place